### PR TITLE
Add synternet

### PR DESCRIPTION
**Pull request by auridasf**

### CODE REVIEW

File: cosmos/synternet.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        "high": 0.03,
+        "low": 0.01
+      }
+    }
+  ],
+  "rest": "https://api.synternet.com",
+  "rpc": "https://rpc.synternet.com",
+  "nodeProvider": {
+    "name": "Synternet",
+    "email": "contact@synternet.com",
```
Comment (auridasf) [inline cosmos/synternet.json]: Not sure what to put here:

email address of the node provider (To help other users reach out when there is an issue with the nodes’ status)

Comment (foxis) [reply cosmos/synternet.json]: Faced the same issue when registering identity in keybase.io. We used `info@synternet.com` there.

Comment (auridasf) [reply cosmos/synternet.json]:
```suggestion
    "email": "info@synternet.com",
```

**Review by foxis — APPROVED** [review]
LGTM

**Pull request merged**